test/tests.ts

```diff
@@ -223,16 +224,22 @@ export function testNitro(
     _handler = await getHandler();
   }, 25_000);
 
-  it("API Works", async () => {
-    const { data: helloData } = await callHandler({ url: "/api/hello" });
-    expect(helloData).to.toMatchObject({ message: "Hello API" });
+  // netlify-edge intentionally ignores prerendered routes
+  it.skipIf(ctx.preset === "netlify-edge")(
```

Comment:
https://github.com/serhalp/nitro/blob/ce72ea3fd1ae59cf722a9868d8e9cc99eb6b0aa1/src/presets/netlify/runtime/netlify-edge.ts#L12-L14

happy to handle this a different way in a follow-up PR if you'd like

Reply:
I would suggest to use unit test for new functionality and avoid any changes here for now.

Reply:
so you'd prefer to split out the addition of the missing tests for netlify-edge into a separate PR?

Reply:
Sure